Misc/NEWS.d/next/Library/2019-01-18-13-44-13.bpo-35537.R1lbTl.rst

```diff
@@ -0,0 +1 @@
+os.posix_spawn() and os.posix_spawnp() now has a setsid parameter.
```

Comment:
I suggest the format:

```suggestion
:func:`os.posix_spawn` and :func:`os.posix_spawnp` now have a *setsid* parameter. 
```

(it's have, not has, no? English is not my first language, I'm not sure :-))